figure/circle.go

```diff
@@ -0,0 +1,17 @@
+package figure
+
+import (
+	"github.com/google/gxui/math"
```

Comment:
Do we really need to use third-party dependency ?
Go has `math` built-in package.

Reply:
Hmm... I think it's because of GoLand autocomplete.
Sure should use regular `Math` here, fixed https://github.com/VSKut/go-temp/pull/8/commits/77d371f251534e99a6df90285313c671917a8705
Thanks for noticing that - I'll keep in mind the possibility of such :)